travis/scala_classpath_for_python.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
+# Compile classes used in python tests
+sbt mleap-spark-extension/compile
+sbt mleap-spark/compile
+
+# Export the complete dependencyClasspath into an env. variable
+export SCALA_CLASS_PATH=\
+"$(sbt --error 'export mleap-spark-extension/runtime:fullClasspath' 2> /dev/null)"
```

Comment:
I was looking for some docs in README on how to run python tests locally. Would that make sense, or are developers supposed to figure it out from this file?

Reply:
Generally, I'd figure out how to run tests from the project setup (follow the steps in a Makefile, or do what tox.ini does). In MLeap we're expected to replicate locally how Travis runs tests, so I'd open the [.travis.yaml](https://github.com/combust/mleap/pull/666/files#diff-354f30a63fb0907d4ad57269548329e3R27) file and read:
```yaml
        - source travis/scala_classpath_for_python.sh
        - SCALA_CLASS_PATH=$SCALA_CLASS_PATH tox -c python/tox.ini -e py36 -v
```
Also the python line `os.env['SCALA_CLASS_PATH']` should trigger a `find` search for `SCALA_CLASS_PATH` in the project.

I generally prefer holding the code accountable for documentation purposes because the code can't go out of date. A README is good until it's up to date.

However, if you feel it could be useful, I'll add a section. I'm only afraid the `.travis.yaml` file could soon change (specially after https://github.com/combust/mleap/pull/665) rendering the README useless. In the meantime, I just noticed I didn't add this in the python Makefile, so I'll add a section there which might also help in documenting this

Reply:
> I generally prefer holding the code accountable for documentation purposes because the code can't go out of date. A README is good until it's up to date.

That's very true. What you described makes sense (open `.travis.yaml` etc.) 👍 

I think for a quick start I would still prefer simple instructions in README. It could be something like this:

----

To compile & run tests:

```
sbt compile test (not sure about this command though?)
```

To run python tests:

- NOTE: `sbt compile` must have been run before this

```
cd python
make (requires some arg?)
```

But if it seems it can't be kept up to date, I won't insist :) It should be ok to find this by checking the travis file.

In any case this shouldn't be a concern for this PR's scope. Thanks for taking the time to explain this to me.

Reply:
So, right now Travis runs Scala tests using this long sequence of commands:
```
  sbt "+ test" "+ mleap-executor-tests/test" "+ mleap-benchmark/test" "mleap-xgboost-runtime/test" "mleap-xgboost-spark/test" "+ publishSigned" "mleap-xgboost-runtime/publishSigned" "mleap-xgboost-spark/publishSigned" "mleap-serving/docker:publish" "mleap-spring-boot/docker:publish"
```
(taken from https://github.com/combust/mleap/blob/master/travis/travis.sh). That might change in #665. I don't think that's in the README.

After my last revision, python tests are as simple as running:
```
make py37_test
```
from the top-level folder. Sourcing is only required if running make from the inner folder